airtable_client.py

```diff
@@ -0,0 +1,36 @@
+from dataclasses import asdict
+import os
+
+from __init__ import app, config
+from airtable import Airtable
+from asset import Asset
+import requests
+
+
+class AirtableClient:
+
+    def __init__(self):
+        """Initialize AirtableClient."""
+
+        self.airtable_client = Airtable(os.getenv("airtable_base_id"),
+                                        config.get("DEFAULT", "table_name"), os.getenv("airtable_api_key"))
+
+    def register_asset(self, register_asset: Asset):
```

Comment:
```suggestion
    def register_asset(self, asset: Asset):
```
引数にメソッド名と同じ名前を使っているのは、紛らわしいです